From 267df77d1d10f6a5134883d5765a718dfb035f72 Mon Sep 17 00:00:00 2001
From: William Wu <wulf@rock-chips.com>
Date: Mon, 27 Mar 2017 16:29:25 +0800
Subject: [PATCH] usb: dwc_otg_310: remove unused configs

The USB20_HOST_EN and USB20_OTG_EN configs are used
for rockchip dwc2 legacy driver, they are not needed
for dwc_otg_310 driver.

Change-Id: I4c16f0be5276b3c07429ab88cb063508b34ce007
Signed-off-by: William Wu <wulf@rock-chips.com>
---
 drivers/usb/dwc_otg_310/Kconfig | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/usb/dwc_otg_310/Kconfig b/drivers/usb/dwc_otg_310/Kconfig
index 95dbe64a32a2..1ad735ef666c 100755
--- a/drivers/usb/dwc_otg_310/Kconfig
+++ b/drivers/usb/dwc_otg_310/Kconfig
@@ -8,10 +8,6 @@ config USB20_HOST
 	  This driver supports Rockchip USB2.0 host
 	  controller.
 
-config USB20_HOST_EN
-	boolean "---usb2.0 host controller enable"
-	depends on USB20_HOST
-
 config USB20_OTG
 	tristate "RockChip USB 2.0 OTG controller"
 	depends on USB || USB_GADGET
@@ -19,10 +15,6 @@ config USB20_OTG
 	  This driver supports Rockchip USB2.0  DWC_OTG
 	  controller.
 
-config USB20_OTG_EN
-	bool "---usb2.0 otg host controller enable"
-	depends on DWC_OTG_HOST_ONLY || DWC_OTG_BOTH_HOST_SLAVE
-
 config DWC_OTG_310
 	tristate
 	default y if USB20_HOST || USB20_OTG
-- 
2.35.3

